script/generateUtilsDoc.sh

```diff
@@ -14,8 +14,26 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+NEW_DEPLOY_NEEDED=0
+
+if [ -z $GITHUB_TOKEN  ]
+then
+    NEW_DEPLOY_NEEDED=0
+    exit 0
+fi
+
 cd $TRAVIS_BUILD_DIR
 
 touch docs/utils.html
 pod2html --infile=lib/utils.pm --outfile=docs/utils.html
 
+#checkout old docs and compare to new ones, then toggle flag accordingly
+git diff gh-pages -- docs/utils.html
```

Comment:
It seems that this won't fly. as these files don't exist inside the gh-pages branch [utils.html](https://github.com/foursixnine/os-autoinst-distri-opensuse/blob/gh-pages/utils.html) does but [docs/utils.html](https://github.com/foursixnine/os-autoinst-distri-opensuse/blob/gh-pages/docs/utils.html) will will not. It only exists on the branch master. You'd need to checkout the file from the git branch, and compare the two. 

Heads's up: 
You would need to check what environment variable will alter the href attribute if the rev="made"  link element in pod2html...
```
diff --git a/docs/utils.html b/utils.html
index 1c4925c35..2573032b0 100644
--- a/docs/utils.html
+++ b/utils.html
@@ -1,10 +1,10 @@
 <?xml version="1.0" ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
-askdj<head>
+<head>
 <title></title>
 <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-<link rev="made" href="mailto:none" />
```